runtime/bajun/src/xcm_config.rs

```diff
@@ -425,11 +428,30 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type VersionWrapper = PolkadotXcm;
 	// Enqueue XCMP messages from siblings for later processing.
 	type XcmpQueue = TransformOrigin<MessageQueue, AggregateMessageOrigin, ParaId, ParaIdToSibling>;
-	type MaxInboundSuspended = sp_core::ConstU32<1_000>;
+	type MaxInboundSuspended = ConstU32<1_000>;
+	type MaxActiveOutboundChannels = ConstU32<128>;
+	// Most on-chain HRMP channels are configured to use 102400 bytes of max message size, so we
+	// need to set the page size larger than that until we reduce the channel size on-chain.
+	type MaxPageSize = ConstU32<{ 103 * 1024 }>;
 	type ControllerOrigin = EnsureRoot<AccountId>;
 	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
-	type WeightInfo = weights::cumulus_pallet_xcmp_queue::WeightInfo<Runtime>;
 	type PriceForSiblingDelivery = NoPriceForMessageDelivery<ParaId>;
+	type WeightInfo = weights::cumulus_pallet_xcmp_queue::WeightInfo<Runtime>;
+}
+
+pub struct BajunActiveXCMChannels;
+
+/// Kusama Basilisk parachain identifier
+pub const BASILISK_PARA_ID: ParaId = ParaId::new(2090);
+
+impl ListChannelInfos for BajunActiveXCMChannels {
+	fn outgoing_channels() -> Vec<ParaId> {
+		vec![BASILISK_PARA_ID]
+	}
+}
+
+impl cumulus_pallet_xcmp_queue::migration::v5::V5Config for Runtime {
+	type ChannelList = BajunActiveXCMChannels;
```

Comment:
Same as in https://github.com/ajuna-network/ajuna-parachain/pull/45#discussion_r1656532600, I suggest merging this PR and I will implement a fix for the migration manually.